.github/workflows/conformance-ingress.yaml

```
@@ -0,0 +1,153 @@
+name: ConformanceIngress
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  pull_request:
+    paths-ignore:
+      - 'Documentation/**'
+      - 'test/**'
+  push:
+    branches:
+      - master
+      - ft/master/**
+    paths-ignore:
+      - 'Documentation/**'
+      - 'test/**'
+
+permissions: read-all
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.event.after }}
+  cancel-in-progress: true
+
+env:
+  minikube_version: 1.25.2
+  cilium_cli_version: v0.11.5
+  timeout: 5m
+  echo_server_image: k8s.gcr.io/ingressconformance/echoserver:v0.0.1@sha256:9b34b17f391f87fb2155f01da2f2f90b7a4a5c1110ed84cb5379faa4f570dc52
+
+jobs:
+  ingress-conformance-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 120
+    steps:
+
+      - name: Install Cilium CLI
+        run: |
+          curl -sSL --remote-name-all https://github.com/cilium/cilium-cli/releases/download/${{ env.cilium_cli_version }}/cilium-linux-amd64.tar.gz{,.sha256sum}
+          sha256sum --check cilium-linux-amd64.tar.gz.sha256sum
+          sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/local/bin
+          rm cilium-linux-amd64.tar.gz{,.sha256sum}
+          cilium version
+
+      - name: Checkout
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          persist-credentials: false
```

Can we move this after the image tag step, and then add a `ref` to the checkout?

```suggestion
      - name: Checkout
        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
        with:
          ref: ${{ steps.vars.outputs.sha }}
          persist-credentials: false
```

This will make no difference for this specific workflow, but it's for consistency with other workflows, e.g.:

https://github.com/cilium/cilium/blob/3b27a8c54210dd35616172b1bd26dab4dcb41c23/.github/workflows/conformance-kind.yaml#L76-L80